src/app/highscores/components/high-score-sidebar/high-score-sidebar.component.html

```diff
@@ -20,10 +20,32 @@
     </button>
 
     <!-- TODO: hide headers -->
-    <mat-tab-group class="header-less-tabs" [selectedIndex]="index">
-      <mat-tab label="Tab 1">Content 1</mat-tab>
-      <mat-tab label="Tab 2">Content 2</mat-tab>
-      <mat-tab label="Tab 3">Content 3</mat-tab>
+    <mat-tab-group class="headerless-tabs" [selectedIndex]="index">
+      <mat-tab>
+        <h1 class="highscore-title">Top Detektive all time!</h1>
+        <table class="highscore-content-table">
+          <thead>
+            <tr>
+              <th>Platz</th>
+              <th>name</th>
+              <th>level</th>
+              <th>experience_points</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr *ngFor="let user of usersList">
```

Comment:
*ngFor has an option to return the index for each element in a list, next to the element itself.

`*ngFor="let user of userList;let index=index;"` 

This way you would not need the new type but could use `{{index + 1}}` instead of `{{user.position}}`